src/redux/slices/attribute-stats-slice.tsx

```diff
@@ -0,0 +1,47 @@
+import { StatsInfo } from "@loaders.gl/i3s";
+
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { RootState } from "../store";
+import { load } from "@loaders.gl/core";
+import { JSONLoader } from "@loaders.gl/loader-utils";
+
+// Define a type for the slice state
+interface statisitcsMapState {
+  /** Statistics map */
+  statisitcsMap: Record<string, StatsInfo | null>;
+}
+
+const initialState: statisitcsMapState = {
+  statisitcsMap: {},
+};
+
+const statisitcsMapSlice = createSlice({
+  name: "statisitcsMap",
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(getAttributeStatsInfo.fulfilled, (state, action) => {
+      state.statisitcsMap[action.payload.statAttributeUrl] =
+        action.payload.stats;
+    });
+  },
+});
+
+export const getAttributeStatsInfo = createAsyncThunk<
+  { stats: StatsInfo | null; statAttributeUrl: string },
+  string
+>("getAttributeStatsInfo", async (statAttributeUrl) => {
+  let stats: StatsInfo | null = null;
+  try {
+    const data = await load(statAttributeUrl, JSONLoader);
+    stats = (data?.stats as StatsInfo) || null;
+  } catch (error) {
+    console.error(error);
+  }
```

Comment:
The error has to be handled with action`getAttributeStatsInfo.rejected`

Reply:
Done